installer/dev/install.cpp

```diff
@@ -157,7 +157,7 @@ namespace ProjectReunionInstaller {
         return outstream;
     }
 
-    void DeployPackageFromResource(const ProjectReunionInstaller::ResourcePackageInfo& resource, const bool quiet)
+    void DeployPackageFromResource(const WindowsAppSDKInstaller::ResourcePackageInfo& resource, const bool quiet)
     {
         // Get package properties by loading the resource as a stream and reading the manifest.
         auto packageStream = GetResourceStream(resource.id, resource.resourceType);
```

Comment:
Line 173, the "PRP" file prefix is an acronym for "Project Reunion Package", so recommend changing this to something like "WAP" or "WAS" or something else more appropriate.

Reply:
Another syntax to hunt for. Thanks

WAS works. I've changed the magic constant to a symbolic constant to aid future reference.